packages/@o3r/design/README.md

```diff
@@ -0,0 +1,47 @@
+<h1 align="center">Otter Design</h1>
```

Comment:
Some documentation on how to use and what kind of code it generates would be nice

Reply:
Created the task #1126 to follow it.